test/owncloud_add_test.cmake

```diff
@@ -21,6 +21,6 @@ macro(owncloud_add_test test_class additional_cpp)
     )
 
     add_definitions(-DOWNCLOUD_TEST)
-    add_definitions(-DOWNCLOUD_BIN_PATH=${CMAKE_BINARY_DIR}/bin)
+    add_definitions(-DOWNCLOUD_BIN_PATH="${CMAKE_BINARY_DIR}/bin")
```

Comment:
This might break on some platforms if the shell eats the quotes when invoking the compiler.

For example http://stackoverflow.com/a/38704229/56315